lib/browser/guest-view-manager.js

```diff
@@ -155,6 +156,10 @@ var createGuest = function (embedder, params) {
   guest.on('size-changed', function (_, ...args) {
     embedder.send.apply(embedder, ['ELECTRON_GUEST_VIEW_INTERNAL_SIZE_CHANGED-' + guest.viewInstanceId].concat(args))
   })
+
+  // Enable DevTools extensions in guest view
+  ChromeExtension.loadDevToolsExtensions(guest)
```

Comment:
We can probably add an API for webContents for hooking WebContents creations and load DevTools Extensions there, instead of injecting this code everywhere:

```javascript
const {webContents} = require('electron')
webContents.on('web-contents-created', function (event, newWebContents) {
  if (newWebContents.getType() !== 'devtools')
    // ...
})
```